### ISSUE TITLE
size mismatch for logits.conv3d.weight   and size mismatch for logits.conv3d.bias

### ISSUE THREAD

**Issue by ducbvh** [dxli94/WLASL#55]
I can't train on colab because it, although have function replace_logit in code. How do i solve this problem

![image](https://user-images.githubusercontent.com/45920660/194005437-70bd16d1-779b-4290-9e0d-847a48be3165.png)

![image](https://user-images.githubusercontent.com/45920660/194005877-56dd80e0-5f0b-4754-89a0-d5e3f3bb45db.png)

**Comment by dxli94** [comment]
Your number of classes is not correctly passed in.

Maybe it is a typo in your command "num_class" --> "num_classes".  I have no experience with the colab you are running so can't be sure.
